src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1378,6 +1378,7 @@ internal enum ReservedAttributes
             NativeIntegerAttribute = 1 << 9,
             CaseSensitiveExtensionAttribute = 1 << 10,
             RequiredMemberAttribute = 1 << 11,
+            LifetimeAnnotationAttribute = 1 << 12,
```

Comment:
>LifetimeAnnotationAttribute = 1 << 12, [](http://example.com/codeflow?start=12&length=38)

I assume we don't want to allow this attribute anywhere in source. In that case, it is probably better to add an explicit check to ```Symbol.DecodeWellKnownAttribute```. More robust and less changes.

Reply:
Especially that it looks like we don't actually perform the check. I think we shouldn't add an infrastructure it, if it is not used. It should be done, if at all, when #62124 will be addressed.